css/main.css

```diff
@@ -228,3 +228,53 @@
 }
 
 /* end of about styling */
+
+/* start of contact and footer styling */
+.contact-section {
+  padding: 120px 24px;
+}
+
+.contact-section-header {
+  color: #172b4d;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 700;
+  text-align: center;
+  margin-bottom: 60px;
+}
+
+.input,
+.input-textarea {
+  width: 100%;
+  padding: 10px;
+  font: 400 15px/24px 'Inter', sans-serif;
+  color: #979493;
```

Comment:
The `color: #979493` seems too deem for the input making user's text seem like placeholders